go.mod

```diff
@@ -3,10 +3,13 @@ module github.com/RHEnVision/provisioning-backend
 go 1.16
 
 require (
+	github.com/aws/aws-sdk-go v1.17.7
```

Comment:
What adds AWS SDK v1? We should only use v2, maybe this is a leftover? Try to do `go mod tidy`.